codelingo.yaml

```diff
@@ -11,7 +11,7 @@ tenets:
   # Overwrite one tenet with custom logic
   # - import: codelingo/effective-go/comment-first-word-when-empty
```

Comment:
The [comment-first-word-when-empty](https://www.codelingo.io/tenets/codelingo/effective-go/comment-first-word-when-empty) tenet was updated back in April to ignore go test files

We can just import this now and remove the override

Reply:
Nice spotting, fixed!